src/test/EigenPod.t.sol

```diff
@@ -366,6 +397,28 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
         cheats.stopPrank();
     }
 
+    function testWithdrawNonBeaconChainETHBalanceWei() public {
+        IEigenPod pod = testDeployAndVerifyNewEigenPod();
+        IEigenPod _pod = eigenPodManager.getPod(podOwner);
+
+        require(_pod == pod, "add check");
+
+        cheats.deal(address(podOwner), 10 ether);
+        emit log_named_address("opd", address(pod));
+
+        (bool sent, ) = payable(address(pod)).call{value: 1 ether}("");
+
+        require(sent == true, "not sent");
+
+        cheats.startPrank(podOwner, podOwner);
+        pod.withdrawNonBeaconChainETHBalanceWei(
+            podOwner,
```

Comment:
Also could add a check that the event is being emitted in `withdrawNonBeaconChainETHBalanceWei`